src/cloud-cli/tests/cli/test_history.py

```diff
@@ -197,37 +164,35 @@ def test_filter_schedule(
         )
 
         assert result.exit_code == 0
-        logged_in.assert_called_with(
-            url=url,
-            params={"page_size": 10, "schedule": "daily"},
+        httpserver.assert_called_with(
+            path=path,
+            query={"page_size": ["10"], "schedule": ["daily"]},
         )
 
     @freeze_time(datetime.datetime(2023, 5, 20, tzinfo=UTC))
     @pytest.mark.parametrize(
         ("lookback", "expected_start_time"),
         (
-            pytest.param("1w", "2023-05-13T00:00:00+00:00", id="1w"),
-            pytest.param("1d", "2023-05-19T00:00:00+00:00", id="1d"),
-            pytest.param("1h", "2023-05-19T23:00:00+00:00", id="1h"),
-            pytest.param("1m", "2023-05-19T23:59:00+00:00", id="1m"),
-            pytest.param("1w2d", "2023-05-11T00:00:00+00:00", id="1w2d"),
-            pytest.param("1w2d3h", "2023-05-10T21:00:00+00:00", id="1w2d3h"),
-            pytest.param("1w2d3h4m", "2023-05-10T20:56:00+00:00", id="1w2d3h4m"),
+            pytest.param("1w", "2023-05-13T00:00:00 00:00", id="1w"),
+            pytest.param("1d", "2023-05-19T00:00:00 00:00", id="1d"),
+            pytest.param("1h", "2023-05-19T23:00:00 00:00", id="1h"),
+            pytest.param("1m", "2023-05-19T23:59:00 00:00", id="1m"),
+            pytest.param("1w2d", "2023-05-11T00:00:00 00:00", id="1w2d"),
+            pytest.param("1w2d3h", "2023-05-10T21:00:00 00:00", id="1w2d3h"),
+            pytest.param("1w2d3h4m", "2023-05-10T20:56:00 00:00", id="1w2d3h4m"),
```

Comment:
aiohttp and werkzeug seem to handle percent-encoding differently so it's better to rely on `pytest-httpserver` matching capabilities: https://github.com/meltano/meltano/pull/7477